js/client-teambuilder.js

```diff
@@ -65,10 +65,12 @@
 				Storage.saveTeams();
 			} else if (this.curSet) {
 				app.clearGlobalListeners();
-				this.curSet = null;
+				Storage.teams[this.curTeamLoc] = Storage.packTeam(this.curTeam);
 				Storage.saveTeams();
+				this.curSet = null;
 			} else if (this.curTeam) {
-				Storage.saveTeam(this.curTeam);
+                Storage.teams[this.curTeamLoc] = Storage.packTeam(this.curTeam);
```

Comment:
Are you commenting on the code functionality or the indentation? 
If the latter, then I don't know what is happening. Everything looks fine in my editor and then when I commit the changes I get stuff like this.